tasks/xgettext.js

```diff
@@ -163,8 +163,8 @@ module.exports = function(grunt) {
           messages = {};
 
         var extractStrings = function(quote) {
-          var regex = /"((?:tooltips|server):::[^"]+)"/g;
-          var subRE = /(tooltips|server):::([^"]+)/g;
+          var regex = /"((?:tooltips):::[^"]+)"/g;
```

Comment:
the non-capture group was needed when there were alternatives. Leaving it in doesn't hurt anything, but it looks a little odd.